root 2020-04-22 20:50:38,325 <module> 24 :
 Driver%3D%7BODBC+Driver+17+for+SQL+Server%7D%3BServer%3Dtcp%3Apydoc-db.database.windows.net%2C1433%3BDatabase%3Dpydoc-db%3BUid%3Dhenry2833%3BPwd%3D%7BInfoip32%7D%3BEncrypt%3Dyes%3BTrustServerCertificate%3Dno%3BConnection+Timeout%3D30%3B

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,357 info 110 :
 SELECT CAST('test plain returns' AS VARCHAR(60)) AS anon_1

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,358 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,359 info 110 :
 SELECT CAST('test unicode returns' AS VARCHAR(60)) AS anon_1

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,360 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,361 info 110 :
 PRAGMA main.table_info("files")

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,361 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,364 info 110 :
 PRAGMA temp.table_info("files")

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,364 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,365 info 110 :
 PRAGMA main.table_info("owner")

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,365 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,366 info 110 :
 PRAGMA temp.table_info("owner")

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,366 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,367 info 110 :
 PRAGMA main.table_info("dates")

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,367 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,367 info 110 :
 PRAGMA temp.table_info("dates")

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,368 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,368 info 110 :
 PRAGMA main.table_info("closure")

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,368 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,369 info 110 :
 PRAGMA temp.table_info("closure")

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,369 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,370 info 110 :
 PRAGMA main.table_info("filename")

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,370 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,371 info 110 :
 PRAGMA temp.table_info("filename")

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,371 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,373 info 110 :
 
CREATE TABLE owner (
	id INTEGER NOT NULL, 
	name VARCHAR(40), 
	PRIMARY KEY (id), 
	UNIQUE (id), 
	UNIQUE (name)
)



sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,374 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,383 info 110 :
 COMMIT

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,386 info 110 :
 
CREATE TABLE files (
	id INTEGER NOT NULL, 
	"fileId" VARCHAR(200), 
	"lastModDate" DATETIME, 
	parent_id INTEGER, 
	"isFile" BOOLEAN, 
	PRIMARY KEY (id), 
	UNIQUE (id), 
	UNIQUE ("fileId"), 
	FOREIGN KEY(parent_id) REFERENCES owner (id), 
	CHECK ("isFile" IN (0, 1))
)



sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,387 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,394 info 110 :
 COMMIT

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,395 info 110 :
 
CREATE TABLE dates (
	id INTEGER NOT NULL, 
	"fileId" INTEGER, 
	moddate DATETIME, 
	CONSTRAINT dates_pk PRIMARY KEY (id), 
	FOREIGN KEY("fileId") REFERENCES files (id)
)



sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,395 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,401 info 110 :
 COMMIT

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,403 info 110 :
 
CREATE TABLE closure (
	id INTEGER NOT NULL, 
	parent INTEGER, 
	child INTEGER, 
	owner_id INTEGER, 
	depth INTEGER, 
	CONSTRAINT closure_pk PRIMARY KEY (id), 
	FOREIGN KEY(parent) REFERENCES files (id), 
	FOREIGN KEY(child) REFERENCES files (id), 
	FOREIGN KEY(owner_id) REFERENCES owner (id)
)



sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,403 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,411 info 110 :
 COMMIT

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,412 info 110 :
 
CREATE TABLE filename (
	id INTEGER NOT NULL, 
	"fileId" INTEGER, 
	"fileName" VARCHAR(600), 
	owner_id INTEGER, 
	CONSTRAINT filename_pk PRIMARY KEY (id), 
	FOREIGN KEY("fileId") REFERENCES files (id), 
	FOREIGN KEY(owner_id) REFERENCES owner (id)
)



sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,413 info 110 :
 ()

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,419 info 110 :
 COMMIT

__main__ 2020-04-22 20:50:38,421 start 87 :
 STARTING SQL testing2020-04-22 20:50:38.421053 /home/henry/pydocs/data/527e4afc-4598-400f-8536-afa5324f0ba4/

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,452 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,454 info 110 :
 INSERT INTO owner (name) VALUES (?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,454 info 110 :
 ('testing2020-04-04 22 Apr0rDzce8QMA',)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,456 info 110 :
 COMMIT

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,467 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,468 info 110 :
 SELECT owner.id AS owner_id, owner.name AS owner_name 
FROM owner 
WHERE owner.id = ?

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,468 info 110 :
 (1,)

__main__ 2020-04-22 20:50:38,469 start 96 :
 Added owner row, name: testing2020-04-04 22 Apr0rDzce8QMA id: 1

__main__ 2020-04-22 20:50:38,603 start 109 :
 Finished loading pickles

__main__ 2020-04-22 20:50:38,605 start 125 :
 File: 1SGPIbZAmFYwIakoHevrYhHMwAf6xNIbMuS1zeULYsTQ

__main__ 2020-04-22 20:50:38,608 start 125 :
 File: 1VJe9pAq5wUkMz1cgNAOYLtvRbDehNWGZLwq4vtX3QVQ

__main__ 2020-04-22 20:50:38,611 start 125 :
 File: 0B4Fujvv5Mfqbb2d0TnNhcFhhREdadmRaaEI3QzVabnE2NE5v

__main__ 2020-04-22 20:50:38,613 start 125 :
 File: 10GJz3CIxCa6CT11Y-kg8f30atim8xSyx

__main__ 2020-04-22 20:50:38,615 start 125 :
 File: 1NpTHMCkLa6lHL9Ok8a9hejyXihpujcQpVgSa7JNBP6s

__main__ 2020-04-22 20:50:38,616 start 125 :
 File: 1OH8h0i2T4Is0R9S8uaZ4C18SQWkxItIW0KwLKnOJGSM

__main__ 2020-04-22 20:50:38,618 start 125 :
 File: 1QNNRjwRnVqieEoA5i9CdaSCyT_6Bi4vkmGO8YuZHytU

__main__ 2020-04-22 20:50:38,620 start 125 :
 File: 1kMTGenwbpJUMiYZMSDbDg_tcROM5USl6

__main__ 2020-04-22 20:50:38,621 start 125 :
 File: 1JEcM-IFEodRvDg7EE9AntQ4wwOBTRXcDFqRK5SR2I08

__main__ 2020-04-22 20:50:38,623 start 125 :
 File: 1LcO1aZmE6Jap4GF5ZpBGetjUSFeVBYg__DgWYlDwWY4

__main__ 2020-04-22 20:50:38,626 start 125 :
 File: 1C09g5-ATBpXNJOmH8ANwCnq7Pk6vClzI1lmFYcqaH6A

__main__ 2020-04-22 20:50:38,628 start 125 :
 File: 1G-K37QbHDgyLsJaUjRsROtLZZkj1fmqyC5YyHzOh-_Q

__main__ 2020-04-22 20:50:38,630 start 125 :
 File: 1cIWwf3nZ2RAeriW8GerxZHY9nirLdcc968-GgSnRkjQ

__main__ 2020-04-22 20:50:38,631 start 125 :
 File: 1yqF_hJ9NyASL9nHpchNsjrb04FUVvTNEmmStnAsh1Sg

__main__ 2020-04-22 20:50:38,633 start 125 :
 File: 16Tz6lbS7tdWBUhgseZCx0va0xBAayzOSSDSH-T20xRA

__main__ 2020-04-22 20:50:38,634 start 125 :
 File: 1leky1DdjFgZcCKZf1EBdNopIRnIdqTLevUE1mliolBI

__main__ 2020-04-22 20:50:38,636 start 125 :
 File: 16wsJnYBwdOHbuGROxWDFe2Tcf1w8T49TwWnGvq5OjXY

__main__ 2020-04-22 20:50:38,637 start 125 :
 File: 1B6ePg9Ixtc4xqKL4D0_JO80n8GL83WpztJKJGaH-6S0

__main__ 2020-04-22 20:50:38,639 start 125 :
 File: 1jRrZTHn-oge2sTqxli8Um026xuniXAeezLZn4B2zVSg

__main__ 2020-04-22 20:50:38,642 start 125 :
 File: 1D48Gz6hmPkE6sATNqYoA3P3Hdjtq1y2zHRjrh1kH64I

__main__ 2020-04-22 20:50:38,645 start 125 :
 File: 1B37ygx-Iwh0EpS1c5fqpbK6AqE39m6ao4WD4uMWknzc

__main__ 2020-04-22 20:50:38,647 start 125 :
 File: 1cctIQ_TyhvRFgFVkzjwcfDL1w0i4L2lF-78_WM7Hy6A

__main__ 2020-04-22 20:50:38,648 start 125 :
 File: 1JQ-JszbkA6JHylaBr1ZDWeMXTy2X6fM2evDEJo-W3U4

__main__ 2020-04-22 20:50:38,650 start 125 :
 File: 11L4quB0D_Ttk0WMkddrcn_r8OmWG5Y7xEWgALTBoNvg

__main__ 2020-04-22 20:50:38,651 start 125 :
 File: 12NnsHOjtevUATb-yYIgDFFLyYajiHxhJ6xHiJnIXj-8

__main__ 2020-04-22 20:50:38,652 start 125 :
 File: 186ge00t4G7kcx1QdCIHxRhTnZfzpSTGDgfAXkdbD_Ig

__main__ 2020-04-22 20:50:38,654 start 125 :
 File: 15T7jFID6g5L9jNgtFSqVnNVFuqL7BqlHq9w9uB3CamE

__main__ 2020-04-22 20:50:38,656 start 125 :
 File: 17MhIJt6Bl2HnDLFiNZPRh7UOx-Co9FMvPWBsRnJRfsM

__main__ 2020-04-22 20:50:38,658 start 125 :
 File: 15e68EIS9xXe5Yvzju4GtkyhFKJ_UgUA9juJKiUpPeUA

__main__ 2020-04-22 20:50:38,660 start 125 :
 File: 1sF1pFtL8fsnQBBAoTt8P7YGOVZVr6-6a4BLeU3PBncI

__main__ 2020-04-22 20:50:38,662 start 125 :
 File: 1G1g252JtmrFnXI5HaLLEafe8apipzquYwQQ0hbQIaho

__main__ 2020-04-22 20:50:38,663 start 125 :
 File: 1wfaH9BSg8wrDaKXR54KK1y8pd-miEOQNj3E8GL6fTfg

__main__ 2020-04-22 20:50:38,665 start 125 :
 File: 10M2HsNol6IOHe6wMXb0WkJV9bmy2445a

__main__ 2020-04-22 20:50:38,666 start 125 :
 File: 1VmB8_0x000GHL4Tbza6LarQMec0Z0esW

__main__ 2020-04-22 20:50:38,668 start 125 :
 File: 1LuJ1Ln_dAc7n3Qj0zNE4yuYfFG6NEgxO-hsDA9M4qpQ

__main__ 2020-04-22 20:50:38,669 start 125 :
 File: 1o6ARVj7jhE2K_VNnX2BYajR2XBX1kdMu

__main__ 2020-04-22 20:50:38,671 start 125 :
 File: 15JWTM6GKGzGaum2h7EvaoJXszPRWFSdINZ4fq6l87LI

__main__ 2020-04-22 20:50:38,673 start 125 :
 File: 15Fi6RHQtC8H-GOJC_gjiED_9rNxtCdahEGtLvBUZcv4

__main__ 2020-04-22 20:50:38,675 start 125 :
 File: 1H1aSubiwctKw-qMV6t1aGlfooY2Qn9V6Y01phHpg-kw

__main__ 2020-04-22 20:50:38,677 start 125 :
 File: 1WbGMwYDyqP6aCDpf5fBbR7sneMicXtvEtfm3Z4k0Vzs

__main__ 2020-04-22 20:50:38,679 start 125 :
 File: 16wh_FgWE8Y0tUpAdIte4WyuB91_ypCySi82lvdhAiFY

__main__ 2020-04-22 20:50:38,680 start 125 :
 File: 1WKk8W5cG-CYKBYN_cWpAdziiPVhvQZEAarnPwf58z4Y

__main__ 2020-04-22 20:50:38,681 start 125 :
 File: 1YJPVyeYstaXiTguoSGCe_Kb3y4yv49N224nUrfLo7rU

__main__ 2020-04-22 20:50:38,683 start 125 :
 File: 14fGVD8kN7ZTYHhthuk5QFTw2j-OnV9QV6BJvuLedIik

__main__ 2020-04-22 20:50:38,684 start 125 :
 File: 17U5RL60qPqJRSKZgj9HpNG0xIgrUI1_buueGhgcZaGs

__main__ 2020-04-22 20:50:38,686 start 125 :
 File: 1PscMFhdtA31Nh7ELmM18NqjUfLDeGiexKI6q-lI0fKY

__main__ 2020-04-22 20:50:38,687 start 125 :
 File: 1YlY2MTwu_I6PGWgmkNqFXYJVvNSQKMi7j0WulyoD75w

__main__ 2020-04-22 20:50:38,689 start 125 :
 File: 1Gxvc-37IOivNHIPUJBI1v3jf5e-Hwcby_VZbN-k-Bfc

__main__ 2020-04-22 20:50:38,691 start 125 :
 File: 11WIU2snHtcamnb9iQszGuvQwhOALT-Gy

__main__ 2020-04-22 20:50:38,693 start 125 :
 File: 1T1Sj69DxCTpEQI2pH0O04l_PSx1V3xsxYOnkiHPEPis

__main__ 2020-04-22 20:50:38,694 start 125 :
 File: 1iwD0UOBkgdB5Jnza1szyk2RCaNpOfito5Be0lD0HGvk

__main__ 2020-04-22 20:50:38,696 start 125 :
 File: 18Fo4vEnjNw9SuVSM0FprUH3bFdl4oVP31YM5H-VH3fs

__main__ 2020-04-22 20:50:38,697 start 125 :
 File: 1NBVW6zOvYcoTmGZ46HZosMmV2SVNURBvf4WOm1ydcTk

__main__ 2020-04-22 20:50:38,698 start 125 :
 File: 1ZBA6pHQ0vfqgMprDjRS0-PqUaeusoF5Z

__main__ 2020-04-22 20:50:38,700 start 125 :
 File: 1VrK4o4hKCSJyT8P4hKL9la_FO7-29hcBJMJDsvjXo2A

__main__ 2020-04-22 20:50:38,701 start 125 :
 File: 1Ez54_TKEcG-xUH_lCEW1711SozJ2_cSeBgiuGKjE8Pk

__main__ 2020-04-22 20:50:38,704 start 125 :
 File: 15oiL5vtc6Pm7TbWGmBPul9ZHEaq5Cp25MmoK7Wj3wQA

__main__ 2020-04-22 20:50:38,706 start 125 :
 File: 1TxO4rhqIX6VUqO2bLJNkukbTpEDNVYSHD0X9hOzi88o

__main__ 2020-04-22 20:50:38,708 start 125 :
 File: 1ZSAyOh8IJqYeY30rg2YNgHU06UxOpmgVqlhKL6puaDs

__main__ 2020-04-22 20:50:38,710 start 125 :
 File: 1kJg8DrOx2wxRXUGvNKc0X_t2GyFMglwXImurNUjTni0

__main__ 2020-04-22 20:50:38,711 start 125 :
 File: 1bVHzlp_4wGefhP9sx5GvI7ZR_ArSbtX0LaFmfZTeB8Y

__main__ 2020-04-22 20:50:38,713 start 125 :
 File: 1c2DcDim6LhJH_JL0iJ23K-KzykdaMUOyvAUnXaAhQEg

__main__ 2020-04-22 20:50:38,714 start 125 :
 File: 1ZNO0oUicsL1By5_92_96mrDU5-FgMK0Dm5bGcYPvUrc

__main__ 2020-04-22 20:50:38,716 start 125 :
 File: 1VJe9pAq5wUkMz1cgNAOYLtvRbDehNWGZLwq4vtX3QVQ

__main__ 2020-04-22 20:50:38,717 start 127 :
 duplicaed fileid found, skipping

__main__ 2020-04-22 20:50:38,719 start 125 :
 File: 1ZSCKTVjGJu6pUx7auNWEyfs83DLJtOeA3AbwuGuY8kI

__main__ 2020-04-22 20:50:38,721 start 147 :
 starting bulk save

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,739 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,739 info 110 :
 ('1SGPIbZAmFYwIakoHevrYhHMwAf6xNIbMuS1zeULYsTQ:22-20.4226430rDzce8QMA', None, 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,741 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,741 info 110 :
 ('1VJe9pAq5wUkMz1cgNAOYLtvRbDehNWGZLwq4vtX3QVQ:22-20.4226430rDzce8QMA', '2020-03-28 12:21:03.616000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,742 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,743 info 110 :
 ('0B4Fujvv5Mfqbb2d0TnNhcFhhREdadmRaaEI3QzVabnE2NE5v:22-20.4226430rDzce8QMA', '2018-01-26 06:14:45.287000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,743 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,744 info 110 :
 ('10GJz3CIxCa6CT11Y-kg8f30atim8xSyx:22-20.4226430rDzce8QMA', '2018-09-14 23:03:28.701000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,745 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,745 info 110 :
 ('1NpTHMCkLa6lHL9Ok8a9hejyXihpujcQpVgSa7JNBP6s:22-20.4226430rDzce8QMA', '2020-02-03 22:42:12.545000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,746 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,746 info 110 :
 ('1OH8h0i2T4Is0R9S8uaZ4C18SQWkxItIW0KwLKnOJGSM:22-20.4226430rDzce8QMA', '2020-01-23 18:51:38.548000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,747 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,747 info 110 :
 ('1QNNRjwRnVqieEoA5i9CdaSCyT_6Bi4vkmGO8YuZHytU:22-20.4226430rDzce8QMA', '2020-03-10 18:28:24.166000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,748 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,748 info 110 :
 ('1kMTGenwbpJUMiYZMSDbDg_tcROM5USl6:22-20.4226430rDzce8QMA', '2019-05-30 05:55:20.970000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,749 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,749 info 110 :
 ('1JEcM-IFEodRvDg7EE9AntQ4wwOBTRXcDFqRK5SR2I08:22-20.4226430rDzce8QMA', '2019-03-12 17:47:04.884000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,750 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,751 info 110 :
 ('1LcO1aZmE6Jap4GF5ZpBGetjUSFeVBYg__DgWYlDwWY4:22-20.4226430rDzce8QMA', '2020-04-06 21:54:16.435000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,751 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,752 info 110 :
 ('1C09g5-ATBpXNJOmH8ANwCnq7Pk6vClzI1lmFYcqaH6A:22-20.4226430rDzce8QMA', '2020-04-10 15:37:31.881000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,752 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,753 info 110 :
 ('1G-K37QbHDgyLsJaUjRsROtLZZkj1fmqyC5YyHzOh-_Q:22-20.4226430rDzce8QMA', '2020-03-26 06:57:29.487000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,753 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,753 info 110 :
 ('1cIWwf3nZ2RAeriW8GerxZHY9nirLdcc968-GgSnRkjQ:22-20.4226430rDzce8QMA', '2020-01-17 18:30:39.243000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,754 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,754 info 110 :
 ('1yqF_hJ9NyASL9nHpchNsjrb04FUVvTNEmmStnAsh1Sg:22-20.4226430rDzce8QMA', '2020-04-15 12:52:38.698000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,755 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,755 info 110 :
 ('16Tz6lbS7tdWBUhgseZCx0va0xBAayzOSSDSH-T20xRA:22-20.4226430rDzce8QMA', None, 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,756 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,756 info 110 :
 ('1leky1DdjFgZcCKZf1EBdNopIRnIdqTLevUE1mliolBI:22-20.4226430rDzce8QMA', '2020-04-06 02:28:19.308000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,757 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,758 info 110 :
 ('16wsJnYBwdOHbuGROxWDFe2Tcf1w8T49TwWnGvq5OjXY:22-20.4226430rDzce8QMA', '2015-07-06 23:38:01.618000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,759 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,759 info 110 :
 ('1B6ePg9Ixtc4xqKL4D0_JO80n8GL83WpztJKJGaH-6S0:22-20.4226430rDzce8QMA', None, 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,760 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,760 info 110 :
 ('1jRrZTHn-oge2sTqxli8Um026xuniXAeezLZn4B2zVSg:22-20.4226430rDzce8QMA', '2020-03-17 14:56:13.451000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,761 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,761 info 110 :
 ('1D48Gz6hmPkE6sATNqYoA3P3Hdjtq1y2zHRjrh1kH64I:22-20.4226430rDzce8QMA', None, 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,761 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,762 info 110 :
 ('1B37ygx-Iwh0EpS1c5fqpbK6AqE39m6ao4WD4uMWknzc:22-20.4226430rDzce8QMA', '2018-05-20 07:33:38.788000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,763 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,763 info 110 :
 ('1cctIQ_TyhvRFgFVkzjwcfDL1w0i4L2lF-78_WM7Hy6A:22-20.4226430rDzce8QMA', '2020-03-27 22:27:31.397000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,763 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,764 info 110 :
 ('1JQ-JszbkA6JHylaBr1ZDWeMXTy2X6fM2evDEJo-W3U4:22-20.4226430rDzce8QMA', None, 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,764 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,764 info 110 :
 ('11L4quB0D_Ttk0WMkddrcn_r8OmWG5Y7xEWgALTBoNvg:22-20.4226430rDzce8QMA', None, 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,765 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,765 info 110 :
 ('12NnsHOjtevUATb-yYIgDFFLyYajiHxhJ6xHiJnIXj-8:22-20.4226430rDzce8QMA', '2020-03-30 18:20:51.634000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,766 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,766 info 110 :
 ('186ge00t4G7kcx1QdCIHxRhTnZfzpSTGDgfAXkdbD_Ig:22-20.4226430rDzce8QMA', '2020-03-06 10:51:33.932000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,767 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,767 info 110 :
 ('15T7jFID6g5L9jNgtFSqVnNVFuqL7BqlHq9w9uB3CamE:22-20.4226430rDzce8QMA', '2020-03-09 01:17:30.970000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,768 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,768 info 110 :
 ('17MhIJt6Bl2HnDLFiNZPRh7UOx-Co9FMvPWBsRnJRfsM:22-20.4226430rDzce8QMA', '2020-03-12 16:48:26.031000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,769 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,769 info 110 :
 ('15e68EIS9xXe5Yvzju4GtkyhFKJ_UgUA9juJKiUpPeUA:22-20.4226430rDzce8QMA', '2020-02-25 15:01:38.567000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,770 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,770 info 110 :
 ('1sF1pFtL8fsnQBBAoTt8P7YGOVZVr6-6a4BLeU3PBncI:22-20.4226430rDzce8QMA', '2018-11-27 09:08:20.070000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,770 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,771 info 110 :
 ('1G1g252JtmrFnXI5HaLLEafe8apipzquYwQQ0hbQIaho:22-20.4226430rDzce8QMA', '2020-02-19 14:32:20.608000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,771 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,772 info 110 :
 ('1wfaH9BSg8wrDaKXR54KK1y8pd-miEOQNj3E8GL6fTfg:22-20.4226430rDzce8QMA', '2020-02-18 11:08:36.897000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,773 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,773 info 110 :
 ('10M2HsNol6IOHe6wMXb0WkJV9bmy2445a:22-20.4226430rDzce8QMA', '2020-02-20 04:49:17.723000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,774 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,774 info 110 :
 ('1VmB8_0x000GHL4Tbza6LarQMec0Z0esW:22-20.4226430rDzce8QMA', '2020-02-21 04:06:55.880000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,775 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,775 info 110 :
 ('1LuJ1Ln_dAc7n3Qj0zNE4yuYfFG6NEgxO-hsDA9M4qpQ:22-20.4226430rDzce8QMA', '2020-02-18 08:54:57.650000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,776 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,776 info 110 :
 ('1o6ARVj7jhE2K_VNnX2BYajR2XBX1kdMu:22-20.4226430rDzce8QMA', '2020-02-18 17:00:26.568000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,777 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,777 info 110 :
 ('15JWTM6GKGzGaum2h7EvaoJXszPRWFSdINZ4fq6l87LI:22-20.4226430rDzce8QMA', '2020-02-15 06:07:44.644000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,777 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,778 info 110 :
 ('15Fi6RHQtC8H-GOJC_gjiED_9rNxtCdahEGtLvBUZcv4:22-20.4226430rDzce8QMA', '2020-02-17 17:20:17.028000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,779 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,779 info 110 :
 ('1H1aSubiwctKw-qMV6t1aGlfooY2Qn9V6Y01phHpg-kw:22-20.4226430rDzce8QMA', '2020-02-17 17:07:27.816000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,779 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,780 info 110 :
 ('1WbGMwYDyqP6aCDpf5fBbR7sneMicXtvEtfm3Z4k0Vzs:22-20.4226430rDzce8QMA', '2020-02-12 10:19:13.585000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,780 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,780 info 110 :
 ('16wh_FgWE8Y0tUpAdIte4WyuB91_ypCySi82lvdhAiFY:22-20.4226430rDzce8QMA', '2020-02-04 13:59:22.043000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,781 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,781 info 110 :
 ('1WKk8W5cG-CYKBYN_cWpAdziiPVhvQZEAarnPwf58z4Y:22-20.4226430rDzce8QMA', '2020-01-28 12:33:02.283000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,782 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,782 info 110 :
 ('1YJPVyeYstaXiTguoSGCe_Kb3y4yv49N224nUrfLo7rU:22-20.4226430rDzce8QMA', '2020-01-28 14:10:20.781000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,784 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,784 info 110 :
 ('14fGVD8kN7ZTYHhthuk5QFTw2j-OnV9QV6BJvuLedIik:22-20.4226430rDzce8QMA', '2020-01-28 21:07:50.805000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,785 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,785 info 110 :
 ('17U5RL60qPqJRSKZgj9HpNG0xIgrUI1_buueGhgcZaGs:22-20.4226430rDzce8QMA', '2019-12-14 00:40:07.307000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,785 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,786 info 110 :
 ('1PscMFhdtA31Nh7ELmM18NqjUfLDeGiexKI6q-lI0fKY:22-20.4226430rDzce8QMA', '2020-01-14 14:19:48.443000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,786 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,787 info 110 :
 ('1YlY2MTwu_I6PGWgmkNqFXYJVvNSQKMi7j0WulyoD75w:22-20.4226430rDzce8QMA', '2020-01-04 11:19:10.128000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,787 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,787 info 110 :
 ('1Gxvc-37IOivNHIPUJBI1v3jf5e-Hwcby_VZbN-k-Bfc:22-20.4226430rDzce8QMA', '2020-01-03 19:01:08.650000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,788 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,788 info 110 :
 ('11WIU2snHtcamnb9iQszGuvQwhOALT-Gy:22-20.4226430rDzce8QMA', '2020-02-02 02:48:39.885000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,790 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,790 info 110 :
 ('1T1Sj69DxCTpEQI2pH0O04l_PSx1V3xsxYOnkiHPEPis:22-20.4226430rDzce8QMA', '2020-01-20 15:01:31.858000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,791 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,791 info 110 :
 ('1iwD0UOBkgdB5Jnza1szyk2RCaNpOfito5Be0lD0HGvk:22-20.4226430rDzce8QMA', None, 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,792 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,793 info 110 :
 ('18Fo4vEnjNw9SuVSM0FprUH3bFdl4oVP31YM5H-VH3fs:22-20.4226430rDzce8QMA', '2020-01-12 00:51:58.994000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,793 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,794 info 110 :
 ('1NBVW6zOvYcoTmGZ46HZosMmV2SVNURBvf4WOm1ydcTk:22-20.4226430rDzce8QMA', None, 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,794 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,795 info 110 :
 ('1ZBA6pHQ0vfqgMprDjRS0-PqUaeusoF5Z:22-20.4226430rDzce8QMA', '2020-01-20 15:15:38.363000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,796 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,796 info 110 :
 ('1VrK4o4hKCSJyT8P4hKL9la_FO7-29hcBJMJDsvjXo2A:22-20.4226430rDzce8QMA', '2020-01-07 20:06:49.774000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,797 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,797 info 110 :
 ('1Ez54_TKEcG-xUH_lCEW1711SozJ2_cSeBgiuGKjE8Pk:22-20.4226430rDzce8QMA', None, 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,798 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,798 info 110 :
 ('15oiL5vtc6Pm7TbWGmBPul9ZHEaq5Cp25MmoK7Wj3wQA:22-20.4226430rDzce8QMA', None, 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,799 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,799 info 110 :
 ('1TxO4rhqIX6VUqO2bLJNkukbTpEDNVYSHD0X9hOzi88o:22-20.4226430rDzce8QMA', '2019-09-28 14:35:20.987000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,800 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,800 info 110 :
 ('1ZSAyOh8IJqYeY30rg2YNgHU06UxOpmgVqlhKL6puaDs:22-20.4226430rDzce8QMA', '2020-01-11 20:44:10.586000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,801 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,801 info 110 :
 ('1kJg8DrOx2wxRXUGvNKc0X_t2GyFMglwXImurNUjTni0:22-20.4226430rDzce8QMA', '2020-01-06 19:03:36.959000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,802 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,802 info 110 :
 ('1bVHzlp_4wGefhP9sx5GvI7ZR_ArSbtX0LaFmfZTeB8Y:22-20.4226430rDzce8QMA', '2019-12-28 16:58:34.677000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,803 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,803 info 110 :
 ('1c2DcDim6LhJH_JL0iJ23K-KzykdaMUOyvAUnXaAhQEg:22-20.4226430rDzce8QMA', None, 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,804 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,804 info 110 :
 ('1ZNO0oUicsL1By5_92_96mrDU5-FgMK0Dm5bGcYPvUrc:22-20.4226430rDzce8QMA', '2019-12-09 14:16:45.695000', 1, 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,805 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:38,805 info 110 :
 ('1ZSCKTVjGJu6pUx7auNWEyfs83DLJtOeA3AbwuGuY8kI:22-20.4226430rDzce8QMA', None, 1, 1)

__main__ 2020-04-22 20:50:38,809 start 149 :
 bulked save done

__main__ 2020-04-22 20:50:38,813 start 174 :
 sess flush done

__main__ 2020-04-22 20:50:39,319 start 184 :
 starting thread for dates

__main__ 2020-04-22 20:50:39,322 commit 43 :
 iters: 72, len: 57935

__main__ 2020-04-22 20:50:39,323 commit 43 :
 iters: 72, len: 57935

__main__ 2020-04-22 20:50:39,323 commit 43 :
 iters: 72, len: 57935

__main__ 2020-04-22 20:50:39,324 commit 43 :
 iters: 72, len: 57935

__main__ 2020-04-22 20:50:39,326 commit 47 :
 flushing, len: 57934

__main__ 2020-04-22 20:50:39,328 commit 47 :
 flushing, len: 57933

__main__ 2020-04-22 20:50:39,328 commit 43 :
 iters: 72, len: 57933

__main__ 2020-04-22 20:50:39,329 commit 43 :
 iters: 72, len: 57933

__main__ 2020-04-22 20:50:39,330 commit 43 :
 iters: 72, len: 57933

__main__ 2020-04-22 20:50:39,330 commit 47 :
 flushing, len: 57932

__main__ 2020-04-22 20:50:39,331 commit 43 :
 iters: 72, len: 57932

__main__ 2020-04-22 20:50:39,331 commit 43 :
 iters: 72, len: 57932

__main__ 2020-04-22 20:50:39,332 commit 43 :
 iters: 72, len: 57932

__main__ 2020-04-22 20:50:39,332 commit 47 :
 flushing, len: 57931

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,360 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,361 info 110 :
 ('0B4Fujvv5Mfqbfmp4aHYzaHBQeHdua3JtUi1ORVFmd3VJbGt4Ylptc0ZXYzNqR0xvWlRRTEk:22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,362 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,362 info 110 :
 ('1A7VHFEYQVj0bigcnBi3pn4NngcNXxH35Croy3zrdvLY:22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,363 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,364 info 110 :
 ('1NO1CVAUsBSgghggdLH2X4YQ2XNaEkDmr7vK156pSCxk:22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,333 commit 43 :
 iters: 72, len: 57931

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,364 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,365 info 110 :
 ('0B4Fujvv5MfqbbW5wLWJIU2JhSTA:22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,335 commit 43 :
 iters: 72, len: 57931

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,366 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,367 info 110 :
 ('root:22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,368 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,336 commit 43 :
 iters: 72, len: 57931

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,368 info 110 :
 ('1WrjmluBbaNBS8u_WZmREl02_oMy2zv6Kcdte82zR5yk:22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,370 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,370 info 110 :
 ('0B4Fujvv5MfqbYjdEcG5iY2tyUTA:22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,337 commit 43 :
 iters: 72, len: 57931

__main__ 2020-04-22 20:50:39,338 commit 43 :
 iters: 72, len: 57931

__main__ 2020-04-22 20:50:39,339 commit 47 :
 flushing, len: 57930

__main__ 2020-04-22 20:50:39,339 commit 43 :
 iters: 72, len: 57930

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,448 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,448 info 110 :
 ('0B4Fujvv5MfqbVElBU01fZUxHcUk22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,449 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,340 commit 47 :
 flushing, len: 57929

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,450 info 110 :
 ('1TYIOfz_uzybWgEeexs8iE64jcjt-AmUegOP6KORNaWQ22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,451 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,452 info 110 :
 ('19Fu317bfo0V7yOIkfLvKtAU-UJYLf8d2Soy-jTvq1uc22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,452 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,453 info 110 :
 ('1eXJKzIcZCM6pAdVLLrvEyIwQBDDrtm8dComeZ_W7uOI22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,453 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,453 info 110 :
 ('1Ux4BA5BVJTcsGcTcGp8CtB85ahlWp0h83AGvrtAhnTg22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,341 commit 43 :
 iters: 72, len: 57929

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,454 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,455 info 110 :
 ('1nOYHEZQN_i8dc6gZNDqguGEmCIU7IX62qngQE0Yvy7o22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,457 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,457 info 110 :
 ('1K6SwyWCOhW6dt906DjOIhJ-ZklpsooCdTyYHGfEZD2s22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,458 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,459 info 110 :
 ('1ak3AnxwEDfv_EkgRG7Q8U_eglT-eK2W3RtAJqlrtcQA22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,342 commit 43 :
 iters: 72, len: 57929

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,460 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,460 info 110 :
 ('1iTV26x-U2-nngY6gONVklMtrhdfI818IwLM682vpABA22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,461 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,462 info 110 :
 ('1gKBPmWO42yparziEkHbvcBtubHyE_IQwtO1Q8IIrW1c22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,463 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,343 commit 43 :
 iters: 72, len: 57929

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,463 info 110 :
 ('0B4Fujvv5MfqbZ3BBZGlsNUtSQTA22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,465 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,344 commit 47 :
 flushing, len: 57928

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,466 info 110 :
 ('12x_9WkC_rJJ4iQW5IUNaY9BwAHRM1P6nRBi_oyRq7eI22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,468 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,468 info 110 :
 ('1FzKITrCYnc_odhNCRNNaZ-PnT7m0f6qI-iDuQG382OY22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,469 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,469 info 110 :
 ('1VzVAOYcQFfJoPSOGAqpK8xA7OtiUNdq13G3SNRtrq2U22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,344 commit 43 :
 iters: 72, len: 57928

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,470 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,471 info 110 :
 ('11VemHYI4DaBG0dTfSBEDkWrOlvYV7B2TxCQkotLFhy822-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,344 commit 43 :
 iters: 72, len: 57928

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,472 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,474 info 110 :
 ('1Xm-pyhQ5D_o4tAHRCtX-amvzVhf-d_pGw0ZhdcTL35Q22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,475 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,476 info 110 :
 ('18s8_aXXl5LtTX5ljUkglb7uf3QgbVuGoTlhwf9iHzKw22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,346 commit 43 :
 iters: 72, len: 57928

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,477 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,477 info 110 :
 ('1V3RqT0n3pc2un7LT3XHJhOLGqmACXSAMSCExEzabbJw22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,478 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,478 info 110 :
 ('1-1y6G4MviqPb0QzFlRjccX3Hlm72FFJS1VppqcHKCTs22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,479 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,479 info 110 :
 ('19yXrTeWC3zjIWztx7RSpKqHL-dpfVf7PPaOmbKyAPrs22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,480 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,481 info 110 :
 ('1HLjWGnWf_XNICjybIlh_C5E1DjjOyLIl8lkXEx8Reu822-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,346 commit 43 :
 iters: 72, len: 57928

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,482 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,482 info 110 :
 ('1KxeMzcqJ7__fOtXnftJ-Pa7r2vlOP2GwzmoDpyt0mBE22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,483 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,484 info 110 :
 ('1Rwm5j7Cnfx80BQ5aCjWLNGe5Zsy2eEIcqnHMhaIgwRc22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,484 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,485 info 110 :
 ('1zyLifTUL_nG8Vm-qDkOCwL5iOSD1DiyDuysUCIgDBXk22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,347 commit 47 :
 flushing, len: 57927

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,486 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,486 info 110 :
 ('0B4Fujvv5MfqbfjRIbkNoSzliOGVEMU1Qdlp6X2RoOTNwZzdtRGhyd2JNRkQwVVh2dnozZGs22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,487 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,347 commit 43 :
 iters: 72, len: 57927

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,487 info 110 :
 ('0B4Fujvv5MfqbfnZ1MnluVHEtTzE3dkxfeGNOS3BOWDB6WE9zSmpGR2pvRDM1TEJRQzRUWm822-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,489 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,348 commit 43 :
 iters: 72, len: 57927

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,490 info 110 :
 ('0B4Fujvv5MfqbflF2aTM2Mnd1QVB3alplUmVSdldWQlpOR0V1OTRXU3M3SXVEd3pFR2FNRVU22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,491 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,491 info 110 :
 ('1cflNpTbjQqpz-E4GqrehbgvXRayQ6dXE_X_x5tC8gbE22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,349 commit 43 :
 iters: 72, len: 57927

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,492 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,493 info 110 :
 ('1BU7kF9C-LzvjiV2WBq5Yx6sstmvvYDDOPKYP8jygvKg22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,495 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,496 info 110 :
 ('1gWhoOIIk2dcQObAvgYpY8Cx_Qc9YUoqNgXEapMic1Qo22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,497 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,497 info 110 :
 ('1xS96CqWG26lxXRjriQhRXyssvP6FimBkTtA5EIOLfao22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,349 commit 47 :
 flushing, len: 57926

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,498 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,499 info 110 :
 ('1qC8O9UkkKeZ82f4iVgf2UYL0ZDqm9ny54I45NmQqBK022-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,500 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,500 info 110 :
 ('14KTZ9MFQGABvko_oR_4LlYTopekxI_fVWLoZQA0N85c22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,500 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,501 info 110 :
 ('0B4Fujvv5MfqbcThuWDlsamUzRWs22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,350 commit 43 :
 iters: 72, len: 57926

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,502 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,503 info 110 :
 ('0B4Fujvv5MfqbMW9TNERCYW1Tc1U22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,503 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,504 info 110 :
 ('0B4Fujvv5MfqbLXBKUEpYMjlUa2c22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,350 commit 43 :
 iters: 72, len: 57926

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,504 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,506 info 110 :
 ('0B4Fujvv5MfqbZXBjWUZzVHM5NHc22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,351 commit 43 :
 iters: 72, len: 57926

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,507 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,508 info 110 :
 ('0B4Fujvv5MfqbX2RMejlDSkc4bHc22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,510 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,510 info 110 :
 ('0B4Fujvv5MfqbQlZtbVBaZ1lReUE22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,351 commit 47 :
 flushing, len: 57925

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,512 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,512 info 110 :
 ('0B4Fujvv5MfqbSVFwbnNzeWFNcjA22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,513 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,352 commit 43 :
 iters: 72, len: 57925

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,514 info 110 :
 ('0B4Fujvv5MfqbZ2JkT01HODJadzQ22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,515 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,365 commit 47 :
 flushing, len: 57924

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,516 info 110 :
 ('0B4Fujvv5MfqbS1dYbGZBNUhFREE22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,517 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,367 commit 47 :
 flushing, len: 57923

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,517 info 110 :
 ('0B4Fujvv5MfqbbFBMeWczTXVKX3M22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,369 commit 47 :
 flushing, len: 57922

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,519 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,519 info 110 :
 ('0B4Fujvv5MfqbY1Q1TjU0LWxhOGM22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,520 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,388 commit 47 :
 flushing, len: 57921

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,521 info 110 :
 ('0B4Fujvv5MfqbQ3RjQW1kUkVHRzg22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,522 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,523 info 110 :
 ('0B4Fujvv5Mfqbd1AtVU1TNmJ6b2822-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,525 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,398 commit 47 :
 flushing, len: 57920

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,526 info 110 :
 ('0B4Fujvv5MfqbUkVUYXBYU0pKdlE22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,450 commit 47 :
 flushing, len: 57919

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,528 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,529 info 110 :
 ('0B4Fujvv5MfqbbVdyZ0dIeFRJNWc22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,530 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,455 commit 47 :
 flushing, len: 57918

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,530 info 110 :
 ('1xYx7Qc0MLZILmxkniIthylVThyLLXr3Gro7pmR6_vUY22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,532 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,532 info 110 :
 ('1b-WTM2ekE1k6pPqFt0v7fa-myJ3Yvhg-vFT_zOs-DA422-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,533 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,533 info 110 :
 ('1UvcAx8uccrerf4uud0w1zvsAsbMSJfyqo298Dh_bq_M22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,534 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,534 info 110 :
 ('1i-Tv07oc_5F72OdEdWf91bbLwoFEaXhmjjlo3QOQyrk22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,535 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,535 info 110 :
 ('12msYuSI-rqvfXROgoGjtxN3Y9AlT3SnRTd6Om5xOxmM22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,536 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,459 commit 47 :
 flushing, len: 57917

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,536 info 110 :
 ('1z0qLmI61do7x0qw3rnK74msvcEZ4tEEVVOERiauTuaI22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,464 commit 47 :
 flushing, len: 57916

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,538 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,539 info 110 :
 ('0B4Fujvv5MfqbfmFlS1dCMGdZSEdWNjF0R1hOUmhrYUIwS0xfY0tuYjkzaE9VLUF5X1VUUUE22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,472 commit 47 :
 flushing, len: 57915

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,540 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,542 info 110 :
 ('1HdWWgJDh4IA7T550gANGtjqDNvq0AwP0serPT4qI34k22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,543 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,475 commit 47 :
 flushing, len: 57914

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,543 info 110 :
 ('0B4Fujvv5MfqbfkVVang5dXd4UGFMOER0a25oWjdIck5tSmRoeExCbUZYUERUU3piWXBMaGc22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,545 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,545 info 110 :
 ('1kzKbaDRw0fxr-yzmfDdTWyiOBuuGprpgdsicAGMD8aw22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,546 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,546 info 110 :
 ('1fVc04agXKNKopxuf3yInvn_EXCPuHxfctQqv9rarZIQ22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,547 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,547 info 110 :
 ('0B4Fujvv5MfqbfnhnOUZ2TTJNeUdIOGtFN2tqMU1ZVlZ6U29KdDRJM05XemJISnpJc2ljV2822-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,477 commit 47 :
 flushing, len: 57913

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,548 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,549 info 110 :
 ('1CEbUajiAYrz2tpwIPZdNCVGniK6leqx04Hmva2N4XM422-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,550 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,550 info 110 :
 ('1Jzh-bWmP8YOEwSm1WsYvN6c8BDgEzMsWsSYXrZ6uwds22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,551 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,482 commit 47 :
 flushing, len: 57912

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,551 info 110 :
 ('1U1hoywM09b-AUO7fIR9TflRSsc3Qu3XmxWmzEph3TmE22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,488 commit 47 :
 flushing, len: 57911

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,552 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,553 info 110 :
 ('1_ovH_D4cpuVTDp9z4DtrqQUeTTjLYI5RWrZ06FXx8fg22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,554 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,490 commit 47 :
 flushing, len: 57910

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,555 info 110 :
 ('1HOWZp6ESjGoYrybdR0LLcpWVlgxys_LqI8yxEeyFkpY22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,494 commit 47 :
 flushing, len: 57909

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,557 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,558 info 110 :
 ('1cX0p4FkT_mRKMngyAqtysHHhT0t8EXJLvv0IYzoS4sU22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,559 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

__main__ 2020-04-22 20:50:39,502 commit 47 :
 flushing, len: 57908

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,560 info 110 :
 ('1WVRprv2UdMlSY1fciu82upcO_aVl7lOnIFAm3XzHZ1Y22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,505 commit 47 :
 flushing, len: 57907

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,562 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,563 info 110 :
 ('1JK8IsrP_wGm2CMPI2TbhvrrYTfzAebH5h56mDeQugoo22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,565 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,565 info 110 :
 ('1bJZoExEi-jg9c7EimHL4Uc3XPLQsrmCZM_Dz_1M5OVg22-20.4226430rDzce8QMA', None, 1, 0)

__main__ 2020-04-22 20:50:39,508 commit 47 :
 flushing, len: 57906

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,566 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,566 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,567 info 110 :
 ('1KfPb3dCRNiKJ-_mj-L5IdTZYQRTQo6xd2jDkCL94W1w22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,569 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,569 info 110 :
 (4, '2018-09-14 23:03:28.701000')

__main__ 2020-04-22 20:50:39,515 commit 47 :
 flushing, len: 57905

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,571 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,572 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,573 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,573 info 110 :
 ('1QCHW4IQHA2vCa1Yoxak6ciYnhE7qlwNW_fnOoSBMEG022-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,573 info 110 :
 (5, '2020-02-03 22:42:18.106000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,575 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,576 info 110 :
 ('1qzyADvsj4voj5Gk1RxlbUhvMUx52JS_4Lz0Rub4nDAI22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,576 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,577 info 110 :
 ('15l9f_tZzB4B4ByKbuD5fJa0bwdk_YqiSmSg62w5TU4E22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,577 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,577 info 110 :
 ('1fb6O0E3ND3FngBsYCfiFvTf5C7xDmIyjFbxCEQGeQK022-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,578 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,578 info 110 :
 ('15A7uqZ5m_7jqIYz5NVUyM0cmStEbSTcezN7qTo0wwXA22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,579 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,579 info 110 :
 ('1aOrKFEEEphoSmQXNpvXYE8OVqfpq4Cm0_pavpJnHNhg22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,579 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,580 info 110 :
 ('1tEY2XbnyEkH5bOQ0y0HmWgJ2vZ5tKsrjh7qZYjdWAyg22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,581 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,581 info 110 :
 ('1WFJYasE9LmtyBf776a2oSGIC_ITRZNZNrHnJbByiHz822-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,581 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,582 info 110 :
 ('19qZnBERiKqL8I8xdBdLdgD7fOFZsNL1NDdcz3TJ9Gkg22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,582 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,583 info 110 :
 ('160R40mTuqkKNjME0oJVZAEM0186W6yAjl0SBUMSIIFY22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,583 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,584 info 110 :
 INSERT INTO files ("fileId", "lastModDate", parent_id, "isFile") VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,584 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,585 info 110 :
 ('0B4Fujvv5MfqbNmZmV0dndDhpUzQ22-20.4226430rDzce8QMA', None, 1, 0)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,585 info 110 :
 (2, '2020-04-16 21:04:51.305000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,602 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,604 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,605 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,606 info 110 :
 (72, 'root', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,606 info 110 :
 (5, '2020-02-03 22:42:19.133000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,608 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,609 info 110 :
 (73, 'Impacts of the Vietnam War', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,609 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,610 info 110 :
 (74, 'Six Traits-A student who hates the teacher', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,610 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,611 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,611 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,612 info 110 :
 (75, 'Writing home', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,613 info 110 :
 (2, '2020-04-16 21:04:56.746000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,613 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,614 info 110 :
 (76, 'Bryan-Should post secondary fees be abolished?', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,614 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,615 info 110 :
 (77, 'transportation', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,615 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,616 info 110 :
 (78, 'sock myth', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,616 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,617 info 110 :
 (79, 'lobes of the brain', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,618 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,618 info 110 :
 (80, 'Medieval Hierarchy', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,619 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,619 info 110 :
 (81, 'Lit Circles', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,620 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,621 info 110 :
 (82, 'Cholera', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,623 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,623 info 110 :
 (83, 'nothing gold can stay.docx', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,624 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,624 info 110 :
 (84, 'Trees.docx', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,625 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,625 info 110 :
 (85, 'Nothing ever happens in my neighborhood', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,626 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,626 info 110 :
 (71, 'School 2014-2015', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,627 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,627 info 110 :
 (86, 'Adaptations to extreme environments', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,627 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,627 info 110 :
 (87, 'Agoraphobia', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,628 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,628 info 110 :
 (88, 'Competition in Education', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,629 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,629 info 110 :
 (89, 'Alexander the Great Pendant', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,630 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,630 info 110 :
 (90, 'Socials Report-Sources', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,630 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,631 info 110 :
 (91, 'Animal Adaptations to living on the ICE', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,631 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,631 info 110 :
 (92, 'Adaptations to extreme environments CONDENSED', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,632 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,632 info 110 :
 (93, 'NOII Article', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,633 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,633 info 110 :
 (94, 'Geography Essay', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,634 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,634 info 110 :
 (95, 'Journalism Symposium', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,635 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,635 info 110 :
 (96, 'Mathematica Centrum', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,635 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,635 info 110 :
 (97, 'Elmacon', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,636 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,636 info 110 :
 (98, 'Term 2', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,637 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,637 info 110 :
 (65, 'Term 1', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,638 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,638 info 110 :
 (99, 'BIG QUESTIONS', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,639 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,639 info 110 :
 (100, 'Sources', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,640 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,640 info 110 :
 (101, 'Cholera affects-political, social, economical', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,641 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,641 info 110 :
 (102, 'Attempts to cure cholera', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,642 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,642 info 110 :
 (103, 'How did cholera affect society', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,643 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,643 info 110 :
 (104, 'SYMPTOMS', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,644 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,644 info 110 :
 (105, 'pyt_eng.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,645 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,645 info 110 :
 (106, 'prep_pyt_eng_2007.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,646 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,646 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,646 info 110 :
 (107, 'Test-prep-Fibonacci-2013-ENG.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,647 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,648 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,648 info 110 :
 (5, '2020-02-03 22:42:22.673000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,648 info 110 :
 (108, 'Test_prep_Pythagore_2012_ENG.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,650 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,650 info 110 :
 (109, 'Test prep Pythagoras 2015 ENG.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,650 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,651 info 110 :
 (110, 'Test prep Pythagoras 2014 ENG.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,651 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,652 info 110 :
 (111, 'Pythagore_prep_2011_ENG.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,653 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,653 info 110 :
 (112, 'pythagoras_eng.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,654 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,654 info 110 :
 (113, '2009Grade6Sprint.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,655 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,655 info 110 :
 (114, '2008Grade6Target.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,656 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,656 info 110 :
 (115, '2008Grade6Sprint.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,657 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,657 info 110 :
 (116, 'C-noanswer.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,658 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,658 info 110 :
 (117, 'B-noanswer.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,658 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,659 info 110 :
 (118, 'A-noanswer.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,659 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,659 info 110 :
 (119, '2009Grade7Sprint.pdf', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,660 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,660 info 110 :
 (120, 'Timeline', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,661 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,661 info 110 :
 (121, 'RCS Contest-Advantages and Disadvantages of being Young', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,662 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,662 info 110 :
 (122, 'Band scripts', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,662 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,663 info 110 :
 (123, 'Ancient Greece thesis', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,663 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,664 info 110 :
 (124, 'Big Questions', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,664 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,665 info 110 :
 (125, 'RP Comment', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,665 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,666 info 110 :
 (126, 'Book Review', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,666 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,666 info 110 :
 (127, 'Letter Home Second Draft', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,667 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,667 info 110 :
 (128, 'Effect of Comics on Chlidren', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,668 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,668 info 110 :
 (129, 'Letter Home Draft 3', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,668 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,669 info 110 :
 (130, 'Science Thesis', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,669 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,669 info 110 :
 (131, 'Daedalus and Icarus', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,670 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,670 info 110 :
 (132, 'Untitled document', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,671 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,671 info 110 :
 (133, '7 important equations', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,672 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,672 info 110 :
 (134, 'Foolproof Instructions', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,673 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,673 info 110 :
 (68, 'Medical Inquiry', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,674 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,674 info 110 :
 (135, 'Three categories of technology', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,675 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,675 info 110 :
 (136, 'Book Review', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,676 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,676 info 110 :
 (137, 'Levels of Competency', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,677 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,677 info 110 :
 (138, 'School Las Vegas vacation', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,678 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,678 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,678 info 110 :
 (139, 'canping squamish school', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,679 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,680 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,680 info 110 :
 (5, '2020-02-03 22:42:19.575000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,681 info 110 :
 (140, 'Book Review-Escape Velocity', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,682 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,683 info 110 :
 (141, 'Book Review-Stormbreaker', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,683 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,683 info 110 :
 (142, 'The Last Invisible Boy-Book Review', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,684 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,685 info 110 :
 (143, 'Book Review #3', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,685 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,685 info 110 :
 (144, 'Draft 4-Final', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,686 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,686 info 110 :
 (145, 'Draft 3', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,687 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,687 info 110 :
 (146, 'Draft 2', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,687 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,688 info 110 :
 (147, 'Draft 1', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,688 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,689 info 110 :
 (148, 'Daedalus and Icarus MASTER', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,690 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,690 info 110 :
 (149, 'Daedalus and Icarus Draft 3', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,691 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,691 info 110 :
 (150, 'Daedalus and Icarus Draft 2', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,692 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,692 info 110 :
 (151, 'X-Ray Tubes', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,693 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,693 info 110 :
 (66, 'Script', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,694 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,694 info 110 :
 (70, 'Future technology draft', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,694 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,695 info 110 :
 (152, 'infographic.png', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,695 info 110 :
 INSERT INTO filename ("fileId", "fileName", owner_id) VALUES (?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,695 info 110 :
 (67, 'Diagnostics Technology', 1)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,708 info 110 :
 COMMIT

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,735 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,738 info 110 :
 SELECT files.id AS files_id, files."fileId" AS "files_fileId", files."lastModDate" AS "files_lastModDate", files.parent_id AS files_parent_id, files."isFile" AS "files_isFile" 
FROM files 
WHERE files.id = ?

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,739 info 110 :
 (65,)

__main__ 2020-04-22 20:50:39,740 commit 56 :
 done flushing

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,743 info 110 :
 SELECT files.id AS files_id, files."fileId" AS "files_fileId", files."lastModDate" AS "files_lastModDate", files.parent_id AS files_parent_id, files."isFile" AS "files_isFile" 
FROM files 
WHERE files.id = ?

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,749 info 110 :
 (68,)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,763 info 110 :
 SELECT files.id AS files_id, files."fileId" AS "files_fileId", files."lastModDate" AS "files_lastModDate", files.parent_id AS files_parent_id, files."isFile" AS "files_isFile" 
FROM files 
WHERE files.id = ?

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,763 info 110 :
 (71,)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,770 info 110 :
 SELECT files.id AS files_id, files."fileId" AS "files_fileId", files."lastModDate" AS "files_lastModDate", files.parent_id AS files_parent_id, files."isFile" AS "files_isFile" 
FROM files 
WHERE files.id = ?

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,771 info 110 :
 (69,)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,775 info 110 :
 SELECT files.id AS files_id, files."fileId" AS "files_fileId", files."lastModDate" AS "files_lastModDate", files.parent_id AS files_parent_id, files."isFile" AS "files_isFile" 
FROM files 
WHERE files.id = ?

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,776 info 110 :
 (67,)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,790 info 110 :
 SELECT files.id AS files_id, files."fileId" AS "files_fileId", files."lastModDate" AS "files_lastModDate", files.parent_id AS files_parent_id, files."isFile" AS "files_isFile" 
FROM files 
WHERE files.id = ?

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,791 info 110 :
 (70,)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,794 info 110 :
 SELECT files.id AS files_id, files."fileId" AS "files_fileId", files."lastModDate" AS "files_lastModDate", files.parent_id AS files_parent_id, files."isFile" AS "files_isFile" 
FROM files 
WHERE files.id = ?

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,795 info 110 :
 (66,)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,803 info 110 :
 SELECT owner.id AS owner_id, owner.name AS owner_name 
FROM owner 
WHERE owner.id = ?

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,803 info 110 :
 (1,)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,813 info 110 :
 INSERT INTO closure (parent, child, owner_id, depth) VALUES (?, ?, ?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,813 info 110 :
 (65, 66, 1, 2)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,854 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,854 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,860 info 110 :
 (2, '2020-04-16 21:04:55.022000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,908 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,920 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:39,931 info 110 :
 (2, '2020-03-28 12:21:07.456000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,015 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,016 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,017 info 110 :
 (2, '2020-03-28 12:21:04.473000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,029 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,030 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,031 info 110 :
 (2, '2020-03-28 12:21:05.187000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,056 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,057 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,057 info 110 :
 (5, '2020-02-03 22:42:20.839000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,123 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,135 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,136 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,142 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,143 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,143 info 110 :
 (2, '2020-03-28 12:21:05.625000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,150 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,171 info 110 :
 (5, '2020-02-03 22:42:14.870000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,177 info 110 :
 (5, '2020-02-03 22:42:12.545000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,201 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,202 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,202 info 110 :
 (5, '2020-02-03 22:42:13.598000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,225 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,236 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,242 info 110 :
 (4, '2020-04-17 03:10:48.927000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,259 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,270 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,281 info 110 :
 (3, '2020-04-17 03:10:48.919000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,286 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,288 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,289 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,292 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,293 info 110 :
 (3, '2018-01-26 06:14:45.287000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,300 info 110 :
 (2, '2020-03-28 12:21:03.616000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,323 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,324 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,324 info 110 :
 (5, '2020-02-03 22:42:23.110000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,326 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,326 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,327 info 110 :
 (5, '2020-02-03 22:42:18.303000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,357 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,358 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,358 info 110 :
 (4, '2018-09-14 23:03:28.945000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,363 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,363 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,364 info 110 :
 (2, '2020-04-16 21:04:55.811000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,392 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,393 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,393 info 110 :
 (3, '2018-01-26 06:14:45.718000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,401 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,402 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,402 info 110 :
 (2, '2020-04-16 21:04:53.939000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,514 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,515 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,515 info 110 :
 (5, '2020-02-03 22:42:21.863000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,527 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,528 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,529 info 110 :
 (2, '2020-03-28 12:21:06.174000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,609 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,610 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,611 info 110 :
 (2, '2020-04-16 21:04:52.691000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,617 info 110 :
 BEGIN (implicit)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,618 info 110 :
 INSERT INTO dates ("fileId", moddate) VALUES (?, ?)

sqlalchemy.engine.base.Engine 2020-04-22 20:50:40,618 info 110 :
 (2, '2020-03-28 12:21:06.659000')

sqlalchemy.engine.base.Engine 2020-04-22 20:50:44,595 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:44,595 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:44,620 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:44,622 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:44,635 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:44,701 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:44,838 info 110 :
 ROLLBACK

root 2020-04-22 20:50:44,849 handle_exception 74 :
 Custom Uncaught exception
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1247, in _execute_context
    self.dialect.do_execute(
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 590, in do_execute
    cursor.execute(statement, parameters)
sqlite3.OperationalError: database is locked

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 193, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/henry/pydocs/processing/sql.py", line 279, in <module>
    start("testing" + datetime.now().__str__(), wpath)
  File "/home/henry/pydocs/processing/sql.py", line 232, in start
    adder(lt_closure, sess)
  File "/home/henry/pydocs/processing/sql.py", line 77, in adder
    sess.flush()
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 2496, in flush
    self._flush(objects)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 2637, in _flush
    transaction.rollback(_capture_exception=True)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 68, in __exit__
    compat.raise_(
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 2597, in _flush
    flush_context.execute()
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/orm/unitofwork.py", line 422, in execute
    rec.execute(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/orm/unitofwork.py", line 586, in execute
    persistence.save_obj(
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/orm/persistence.py", line 239, in save_obj
    _emit_insert_statements(
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/orm/persistence.py", line 1135, in _emit_insert_statements
    result = cached_connections[connection].execute(
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 984, in execute
    return meth(self, multiparams, params)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/sql/elements.py", line 293, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1097, in _execute_clauseelement
    ret = self._execute_context(
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1287, in _execute_context
    self._handle_dbapi_exception(
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1481, in _handle_dbapi_exception
    util.raise_(
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1247, in _execute_context
    self.dialect.do_execute(
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 590, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) database is locked
[SQL: INSERT INTO closure (parent, child, owner_id, depth) VALUES (?, ?, ?, ?)]
[parameters: (65, 66, 1, 2)]
(Background on this error at: http://sqlalche.me/e/e3q8)
sqlalchemy.engine.base.Engine 2020-04-22 20:50:44,890 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:44,960 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,040 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,062 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,096 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,199 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,200 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,229 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,237 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,273 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,314 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,329 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,329 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,347 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,348 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,378 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,384 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,414 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,424 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,533 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,550 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,631 info 110 :
 ROLLBACK

sqlalchemy.engine.base.Engine 2020-04-22 20:50:45,637 info 110 :
 ROLLBACK

root 2020-04-22 20:50:47,128 handle_exception 78 :
 exiting! from sshook

root 2020-04-22 20:50:47,130 handle_exception 80 :
 sending to default hook

sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,172 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563164669696 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,178 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd7a0031c60>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563164669696 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563164669696 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,183 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562636191488 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,186 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd7a0031d50>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562636191488 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562636191488 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,191 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563785434880 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,194 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd7a0031e40>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563785434880 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563785434880 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,198 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562619406080 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,200 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c8030>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562619406080 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562619406080 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,205 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563198240512 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,209 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c8120>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563198240512 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563198240512 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,214 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562132891392 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,216 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c8300>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562132891392 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562132891392 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,220 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563760256768 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,224 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c83f0>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563760256768 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563760256768 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,228 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563793827584 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,231 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c84e0>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563793827584 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563793827584 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,235 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140564038047488 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,240 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c85d0>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140564038047488 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140564038047488 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,244 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562107713280 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,247 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c8210>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562107713280 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562107713280 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,250 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140564029654784 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,254 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c86c0>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140564029654784 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140564029654784 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,259 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562124498688 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,262 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c87b0>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562124498688 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562124498688 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,267 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140564021262080 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,269 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c88a0>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140564021262080 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140564021262080 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,274 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562644584192 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,277 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c8a80>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562644584192 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562644584192 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,280 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562661369600 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,284 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c8990>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562661369600 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562661369600 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,288 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562652976896 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,292 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c8b70>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562652976896 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562652976896 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,296 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563156276992 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,300 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c8c60>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563156276992 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563156276992 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,305 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563181455104 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,308 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c8d50>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563181455104 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563181455104 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,313 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563189847808 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,316 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7e7030>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563189847808 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563189847808 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,321 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140564046440192 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,325 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7c8e40>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140564046440192 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140564046440192 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,329 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562099320576 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,331 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7e7120>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562099320576 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562099320576 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,336 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562627798784 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,340 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7e7210>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562627798784 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562627798784 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,343 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562669762304 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,346 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7e7300>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562669762304 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562669762304 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,349 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563206633216 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,352 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7e73f0>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563206633216 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563206633216 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,358 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563173062400 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,361 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7e74e0>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563173062400 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563173062400 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,365 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563768649472 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,368 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7e75d0>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563768649472 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563768649472 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,372 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562116105984 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,375 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7e76c0>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562116105984 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140562116105984 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,380 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563810612992 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,384 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7e77b0>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563810612992 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563810612992 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,388 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563777042176 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,391 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7e78a0>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563777042176 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563777042176 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,395 _finalize_fairy 701 :
 Exception during reset or similar
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563802220288 and this is thread id 140564704171840.
sqlalchemy.pool.impl.NullPool 2020-04-22 20:50:47,397 _close_connection 272 :
 Exception closing connection <sqlite3.Connection object at 0x7fd78c7e7990>
Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 693, in _finalize_fairy
    fairy._reset(pool)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 880, in _reset
    pool._dialect.do_rollback(self)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563802220288 and this is thread id 140564704171840.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 270, in _close_connection
    self._dialect.do_close(connection)
  File "/home/henry/.virtualenvs/pydocs/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 546, in do_close
    dbapi_connection.close()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140563802220288 and this is thread id 140564704171840.
